sass/theme/_gallery_page.scss

```diff
@@ -0,0 +1,68 @@
+$container-padding: 0px !important;
+
+.current-gallery {
+  // todo ask marianne: how do we best make this snap to full screen or something on mobile? setting
```

Comment:
going to merge this PR cause it looks pretty good, tomorrow I can do a day on mobile compatibility or something, look into this + anything George may have missed